model/enduser/registry.yaml

```diff
@@ -0,0 +1,18 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes information about the end user, which can be used as a subdomain of browser, client, or user domains.
+    attributes:
+      - id: enduser.pseudo.id
+        type: string
+        stability: experimental
+        brief: >
+          Pseudonymous identifier of an end user. This identifier is unique to the user but does not reveal their actual identity.
+        examples: ['QdH5CAWJgqVT4rOr0qtumf']
+      - id: enduser.authentication.id
+        type: string
+        brief: "Unique identifier of an authenticated user in the system."
+        examples: [ 'S-1-5-21-202424912787-2692429404-2351956786-1000' ]
```

Comment:
it seems to be an example of windows OS user ([SID](https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-identifiers)). Given that this attribute is intended to represent human identity, we should use a realistic example (login? email? guid?)

Also, we should add a note that it's likely a PII. I have a proposal on how to capture it (until we have a formal way in #1707)

https://github.com/open-telemetry/semantic-conventions/blob/d09cce343377962be7d6afe72d71d39081df3630/docs/general/how-to-define-semantic-conventions.md?plain=1#L70-L76